### PR TITLE
OKTA-546072: add enrollAuthenticators description

### PR DESCRIPTION
**Pull request by jonathanhansen-okta**
## Description:
* enrollAuthenticators field for Profile Enrollment Policy. This field will determine the authenticators that should be used on the first page of user registration
* Is this PR related to a Monolith release? Yes. 2022.11.0

### Resolves:

* [OKTA-546072](https://oktainc.atlassian.net/browse/OKTA-546072)

### CODE REVIEW

**Review by zeusandrade-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -2234,5 +2235,6 @@ Policy Rule conditions aren't supported for this policy.
...
 | `unknownUserAction`          | Which action should be taken if this User is new (Valid values: `DENY`, `REGISTER`)                                               | String  | YES | N/A                                                                                                                                                                                                                        |
 | `uiSchemaId`                 | Value created by the backend. If present all policy updates must include this attribute/value.                                               | String  | Required if Present | N/A                                                                                                                                                                                                                        |
+| `enrollAuthenticators` | Additional authenticator fields that should be used on the first page of user registration (Valid values: `password`) | Array | No | N/A |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2238]: If this property isn't required, then I think we should soften the language here:
```suggestion
| `enrollAuthenticators` | Additional authenticator fields that can be used on the first page of user registration (Valid values: `password`) | Array | No | N/A |
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM! just one nit

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md (hunk 1 above)
Comment (jonathanhansen-okta) [reply packages/@okta/vuepress-site/docs/reference/api/policy/index.md]: Done

**Pull request merged**